workshops/forms.py

```diff
@@ -393,17 +391,10 @@ def clean_end(self):
 
     class Meta:
         model = Event
-        # reorder fields, don't display 'deleted' field
         fields = ('slug', 'completed', 'start', 'end', 'host', 'administrator',
-                  'assigned_to', 'tags', 'url', 'language', 'reg_key',
+                  'assigned_to', 'tags', 'url', 'language', 'reg_key', 'venue',
                   'admin_fee', 'invoice_status', 'attendance', 'contact',
-                  'notes', 'country', 'venue', 'address', 'latitude',
-                  'longitude', 'learners_pre', 'learners_post',
-                  'instructors_pre', 'instructors_post', 'learners_longterm')
-
-        # WARNING: don't change put any fields between 'country' and
-        #          'longitude' that don't relate to the venue of the event
-
+                  'notes', 'country', 'address', 'latitude', 'longitude', )
```

Comment:
This seems like a violation of DRY. I am repeating the fields I would to display in both `helper.layout` and `Meta.fields`.